device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,82 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 interface connected to the DUT.
+            This name must match the "name" field of the "ipv4_addresses" on top which this RSVP interface is configured.          
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
+          description: >-
+            IPv4 address of the RSVP neighbor on this interface.
+          type: string
+          format: ipv4
+        label_space_start:
+          description: >-
+            The user-defined label space start value. The LSPs for which this router acts as a egress are assigned labels from this
+            label pool.The"label_space_start" and "label_space_end" together defines this label-pool. 
+          type: integer
+          default: 1000
+          minimum: 0
+          maximum: 1048575
+        label_space_end:
+          description: >-
+            The user-defined label space end value.The last label value that can be assigned to the LSPs for which this router acts as egress.
+          type: integer
+          default: 100000
+          minimum: 0
+          maximum: 1048575
+        enable_refresh_reduction:
+          description: >-
+            Enables sending of Refresh Reduction as described in RFC2961.
+          type: boolean
+          default: false
+        summary_refresh_interval:
+          description: >-
+            The number of seconds between transmissions of successive Summary Refreshes.
+            There is no specification specified maximum value. For clarity, setting the maximum to 1 hour.
+          type: integer          
+          default: 30
+          minimum: 1
+          maximum: 3600
+        send_bundle:
+          description: >-
+            Enables aggregration of different RSVP messages within a single PDU.
+          type: boolean
+          default: false
+        bundle_threshold:
+          description: >-
+            The number of milliseconds to wait after which RSVP will bundle different RSVP messages and transmit Bundle messages.
+          type: integer
+          default: 50
+          minimum: 20
+          maximum: 1000
+        enable_hello:
+          description: >-
+            Enables sending of Hello Messages as per RFC3209.            
+          type: boolean
+          default: false
+        hello_interval:
```

Comment:
have we checked major NOSes for what their default values are that might make sense?

Reply:
based on open config using 9 sec.Some of the Duts and IxNetwork use 10 sec as default. Lines 170 to 175 in
https://github.com/openconfig/public/blob/master/release/models/mpls/openconfig-mpls-rsvp.yang

    leaf hello-interval {
    type uint16 {
        range 1000..60000;
      }
      units milliseconds;
      default 9000;